addons/web-support/api/src/main/java/org/jboss/windup/web/addons/websupport/rest/graph/applicationDetails/ApplicationMessageReducedDTO.java

```diff
@@ -0,0 +1,50 @@
+package org.jboss.windup.web.addons.websupport.rest.graph.applicationDetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.enterprise.inject.Vetoed;
+
+/**
+ * Contains messages that apply at the application level.
+ *
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+@Vetoed
+@JsonIgnoreProperties({"handler", "delegate"})
```

Comment:
Yes, these get proxied by Furnace, so they get the additional properties from a Java proxy object. Jackson will then try to serialize these and it often introduces cycles.

I haven't found an elegant solution for it.